### DISCUSSION TITLE
Changeset on dependabot

### DISCUSSION THREAD

**Discussion by Noggling** [equinor/fusion-project-portal#410] [General]
Today, the "create changeset" action is triggered on `Dependabot` pull requests, but it encounters issues and fails. Should we consider implementing the functionality of generating a default changeset when specific conditions are met? Additionally, should we use the pull request title as changeset description?

**Comment by Noggling** [comment]
This also applies to SNYC pull requests 😮‍💨

**Comment by Noggling** [comment]
@kjetilhau @AreWiv @Jossilainen @ken-mellem 
Is this necessary for the end-user to know when we update dependencies? if not we can just add a simple skipp check.

**Comment by ken-mellem** [comment]
Is this still relevant?
